src/components/Pagination.vue

```diff
@@ -84,10 +84,17 @@ export default {
 
       return pages;
     },
+    totalResults() {
+      return this.searchStore.totalResults;
+    },
   },
   methods: {
     goToPage(page) {
-      this.searchStore.page = Math.min(this.totalPages, page);
+      let p = page;
```

Comment:
I implemented your style suggestion.
The pagination starts at 1 in instantsearch libraries so I made this library do the same for consistency purposes.